core/src/test/java/com/klaytn/caver/common/KeyringContainerTest.java

```diff
@@ -228,14 +226,16 @@ public void updateToCoupledKeyring() {
         //CA-KEYRINGCONTAINER-008
         @Test
         public void updateToDecoupledKeyring() {
-            KeyringContainer container = new KeyringContainer();
-            SingleKeyring coupled = KeyringFactory.generate();
-            SingleKeyring deCoupled = KeyringFactory.createWithSingleKey(coupled.getAddress(), PrivateKey.generate().getPrivateKey());
+            SingleKeyring coupled = caver.wallet.keyring.generate();
+            SingleKeyring deCoupled = caver.wallet.keyring.createWithSingleKey(
+                    coupled.getAddress(),
+                    caver.wallet.keyring.generate().getKey().getPrivateKey()
```

Comment:
I think you can use `caver.wallet.keyring.generateSingleKey(1)` here :)

Reply:
That would be more great :)

I checked [caver-js/test/packages/caver.wallet#L184](https://github.com/klaytn/caver-js/blob/ae1b51295614131c364b1e3e16a82a98bb629e2e/test/packages/caver.wallet.js#L184) is creating a private key string by calling `caver.wallet.keyring.generateSingleKey()` which is also implemented in caver-java.

Thanks for your feedback.